CriticalMass/ChatInputViewController.xib

```diff
@@ -0,0 +1,90 @@
+                            <rect key="frame" x="0.0" y="4" width="344" height="38"/>
+                            <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                            <accessibility key="accessibilityConfiguration" hint="enter chat message in this view" label="chat message input "/>
+                            <constraints>
+                                <constraint firstAttribute="height" priority="999" constant="38" placeholder="YES" id="kX9-Co-fmB"/>
+                            </constraints>
+                            <edgeInsets key="layoutMargins" top="0.0" left="16" bottom="0.0" right="50"/>
+                            <color key="textColor" systemColor="systemIndigoColor" red="0.34509803919999998" green="0.33725490200000002" blue="0.83921568629999999" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                            <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                            <textInputTraits key="textInputTraits" keyboardAppearance="alert" returnKeyType="send"/>
+                            <userDefinedRuntimeAttributes>
+                                <userDefinedRuntimeAttribute type="number" keyPath="cornerRadius">
+                                    <real key="value" value="19"/>
+                                </userDefinedRuntimeAttribute>
+                                <userDefinedRuntimeAttribute type="color" keyPath="placeholderColor">
+                                    <color key="value" white="0.66666666666666663" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                </userDefinedRuntimeAttribute>
+                            </userDefinedRuntimeAttributes>
+                            <connections>
+                                <outlet property="delegate" destination="-1" id="lNh-fE-Jtu"/>
+                            </connections>
+                        </textView>
+                        <button opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" enabled="NO" contentHorizontalAlignment="center" contentVerticalAlignment="center" adjustsImageSizeForAccessibilityContentSizeCategory="YES" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="ok1-Cv-1uN" customClass="SendButton" customModule="CriticalMaps" customModuleProvider="target">
+                            <rect key="frame" x="352" y="4" width="38" height="38"/>
+                            <accessibility key="accessibilityConfiguration" hint="Sends the chat message" label="send message button"/>
```

Comment:
the accessibilityLabel shouldn't contain the elements trait "button" as this gets already announced by VoiceOver and it needs to be translated

Reply:
just "send" should be enough as a Label

Reply:
Thanks. I am a little unsure what is the best practice here